src/spine.js

```diff
@@ -1,3 +1,7 @@
+// ./wrapper.js will be aliased to other versions during builds
+import { spine } from './wrapper.js';
+import * as semver from './semver/index.js';
```

Comment:
Can we simply import `semver` from `node_modules` now?

Reply:
That is.  That selection wrapper imports from semver, saves 20 KB on the build (see top of semver readme), and is needed for rollup/node_resolve to do its thing.

But that brings up another point, I can drop the commands for pre-generated semver iife and the files in contrib.

Reply:
completed in: 292b754